app/views/propertyLinking/relationshipToProperty.scala.html

```diff
@@ -0,0 +1,115 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.ApplicationConfig
+@import controllers.propertyLinking.ClaimPropertyRelationshipVM
+@import helpers._
+
+@this(govukErrorSummary : GovukErrorSummary, govukDetails : GovukDetails, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout)
+@(model: ClaimPropertyRelationshipVM, clientDetails: Option[ClientDetails] = None, backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@clientId = @{clientDetails.map(_.organisationId)}
+@form = @{model.form}
+@clientSuffix = @{
+    clientId match {
+        case Some(id) => { ".client"}
+        case _        => {""}
+    }
+}
+@clientParameter = @{
+    clientDetails  match {
+        case Some(client) => { s"?organisationId=${client.organisationId}&organisationName=${client.organisationName}"  }
+        case _        => ""
+    }
+}
+
+
+@getEnrichedErrorLinks = @{
```

Comment:
why do we need this?

Reply:
We need this to Display custom error messages